src/panels/profile/ha-refresh-tokens-card.ts

```diff
@@ -35,6 +35,18 @@ class HaRefreshTokens extends LitElement {
       refreshTokens?.filter((token) => token.type === "normal").reverse()
   );
 
+  compareTokenLastUsedAt = (tokenA: RefreshToken, tokenB: RefreshToken) => {
```

Comment:
you can move is outside of the class, as it doesn't use any class properties.
```suggestion
  const compareTokenLastUsedAt = (tokenA: RefreshToken, tokenB: RefreshToken) => {
```